docs/DeveloperGuide.md

```diff
@@ -132,6 +132,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
+### [In Progress] \[DateTime\]
+
+#### Implementation
+The implementation allows users to parse and compare unique DateTimes. 
+
+#### Parsing
```

Comment:
Standardize headings - we are probably only sticking to Implementation and Design Consideration